server/db/db-query.js

```diff
@@ -2,23 +2,25 @@ const { Client } = require("pg");
 require("dotenv").config();
 
 const logQuery = (statement, parameters) => {
-  let timeStamp = new Date();
-  let formattedTimeStamp = timeStamp.toString().substring(4, 24);
+  const timeStamp = new Date();
+  const formattedTimeStamp = timeStamp.toString().substring(4, 24);
   console.log(formattedTimeStamp, statement, parameters);
```

Comment:
I'm curious what this `logQuery` function's purpose is since there's no return value, Is it just for debugging?

Reply:
It's purely for logging/debugging. It's a relic of the "todo" app code I copied this from.